test/README.md

```diff
@@ -309,6 +309,18 @@ identified and with the mutual understanding that PRs related to
 that RHBZ must include an automated test reproducer.
 
 
+## QE associates/responsibilities matrix
+
+- [@atodorov](https://github.com/atodorov) - QE lead;
+  osbuild, osbuild-composer, weldr-client, composer-cli, image-builder, image-builder-frontend,
+  bug testing & verification, anything else
+- [@jrusz](https://github.com/jrusz) - testing across the board; backup for atodorov;
+  osbuild, osbuild-composer, weldr-client, composer-cli, image-builder, image-builder-frontend,
+  bug testing & verification, anything else
+- [@henrywang](https://github.com/henrywang) - everything related to testing RHEL for Edge images
```

Comment:
@atodorov Could you please add @yih-redhat in this list? He's another RHEL for Edge project QE. Thanks.

[@yih-redhat](https://github.com/yih-redhat) - everything related to testing RHEL for Edge images; backup for @henrywang

Reply:
Added. It probably makes sense to also add them to our QE meeting. See your updated invite for a new timeslot which serves better your timezone.